xclim/sdba/utils.py

```diff
@@ -416,8 +421,13 @@ def _interp_quantiles_1D(newx, oldx, oldy):
     # else:
 
     def _interp_quantiles_2D(newx, newg, oldx, oldy, oldg):
+        print('|')
```

Comment:
remove

Reply:
Argh, now you see my secret debugging methods!